client/SDL/sdl_freerdp.cpp

```diff
@@ -1253,6 +1255,20 @@ int main(int argc, char* argv[])
 	return rc;
 }
 
+BOOL SdlContext::disconnect_session()
+{
+
+	std::lock_guard<CriticalSection> lock(critical);
+	for (uint32_t x = 0; x < 1; x++)
```

Comment:
just call `freerdp_abort_connect_context` here? if the rdp thread terminates it will clean up `SDL` context properly.

Reply:
I'll make the change.